lib/sidebar_pages.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: false
+
+require 'forwardable'
+
+module SwedbankPay
+  # Provides filtering and utility methods on the pages used to build the Sidebar
```

Comment:
Codacy found an issue: [Line is too long. [81/80]](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6198407)